packages/vitest/src/integrations/chai/index.ts

```diff
@@ -22,6 +22,9 @@ export function createExpect(test?: TaskPopulated) {
     else
       return assert
   }) as ExpectStatic
+
+  // quick and dirty way to share custom asymmetric matchers between global expect and local expect
+  Object.assign(expect, (globalThis as any).__hackAsymmetricMatchers ??= {})
```

Comment:
`__hackAsymmetricMatchers` should be defined in `expect` package somewhere (probably inside a `state.ts` file), otherwise others using the package will get an error